base/bucket_n1ql.go

```diff
@@ -233,3 +235,9 @@ func QueryCloseErrors(closeError error) []error {
 	return closeErrors
 
 }
+
+func SlowQueryLog(startTime time.Time, messageFormat string, args ...interface{}) {
+	if elapsed := time.Now().Sub(startTime); elapsed > SlowQueryWarningThreshold {
+		Infof(KeyQuery, messageFormat+" took "+elapsed.String(), args...)
```

Comment:
I think this needs to be run through `gofmt`.

Reply:
That's already been through gofmt (go 1.8.5).